custom-widget-builder/src/CustomWidgetBuilder.ts

```diff
@@ -204,25 +204,6 @@ export class CustomWidgetBuilder {
     return properties;
   }
 
-  private static getAssets(wcId: string, wcBundle: string): Array<Asset> {
```

Comment:
Why remove assets method ? 
Some widget can contains assets

Reply:
As discussed, for now all the widgets assets are in the web component bundle.